.github/workflows/CI.yml

```diff
@@ -18,7 +18,6 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - '1.6'
           - '1.7'
```

Comment:
Best to change this to `'1'`, which automatically fills in the latest released version (which currently is 1.7, but then will switch to 1.8 when that is released).